Dockerfile.astera

```
@@ -26,6 +26,35 @@ ENV DEBIAN_FRONTEND=noninteractive \
+# image ships git 2.34.1, where only an exact path or the bare `*` is honoured —
+# prefix globs are silently ignored (verified). The cache paths are
+# content-hashed, so exact entries are whack-a-mole, and `*` would disable the
+# ownership check on genuinely shared volumes like /mnt/diffuse-shared too.
+#
+# pixi already redirects its repodata and pypi-mapping caches off network
+# filesystems on its own; it just does not do the same for the uv cache. Setting
+# the cache root to container-local disk covers all of them. The trade-off is
+# that the download cache no longer survives a pod restart — environments
+# themselves live in the synced checkout and are unaffected.
+RUN mkdir -p /var/cache/pixi
     SAMPLEWORKS_SKIP_ENV_PREPARE=1
```

Hadolint (2.15.1)

[error] unexpected 'S'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)